src/client/Components/Util/StatTable.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export default class StatTable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      github_repo: this.props.github_repo,
+      isLoaded: false
+    };
+  }
+
+  componentDidMount() {
+    this.getComponentStats();
+  }
+
+  getComponentStats() {
+    fetch(`http://v9_website.ngrok.io/api/db/getComponentStatus?component=${this.state.github_repo}`)
+      .then(res => res.json())
+      .then(
+        (result) => {
+          this.setState({
+            isLoaded: true,
+            componentStats: result.reverse()
+          });
+        }
+      );
+  }
+
+  render() {
+    if (!this.state.isLoaded) {
+      return (<p />);
+    }
```

Comment:
Maybe should display a bar or something?

Reply:
While you're probably not wrong, doesn't seem to make sense to adjust design things if we're completely changing it for the next upgrade.